server/plugin.go

```diff
@@ -70,6 +66,25 @@ func (p *Plugin) updateConfig(f func(conf *config)) config {
 	return p.conf
 }
 
+// saveConfigToServer persists the externalConfig portion of the plugin config to the server's config.json
```

Comment:
Is this function still needed?

Reply:
No, it's not used, I hesitated to delete it because we will need it in the future. I'll get rid of it here and keep it on file for later.